spec/ttml1.xml

```diff
@@ -6654,9 +6654,6 @@ ref="semantics-style-inheritance-content"/> below.</p>
 other out-of-line style properties, thus creating a chain of references starting at the
 affected element. When expressed in this manner, the association of style information
 is referred to as <emph>chained referential styling</emph>.</p>
-<p>If the same style property is specified in more than one referenced
```

Comment:
I am opposed to removing this. If there is a problem with it, then let's fix it.